lib/Language/Souffle/Interpreter.hs

```diff
@@ -0,0 +1,237 @@
+
+{-# OPTIONS_GHC -Wno-redundant-constraints #-}
+{-# LANGUAGE RankNTypes, FlexibleInstances, FlexibleContexts, DataKinds #-}
+{-# LANGUAGE ScopedTypeVariables, TypeFamilies, TypeOperators #-}
+{-# LANGUAGE DerivingVia, InstanceSigs, UndecidableInstances, BangPatterns #-}
+{-# LANGUAGE GeneralisedNewtypeDeriving, DefaultSignatures, DeriveGeneric, LambdaCase #-}
+{-# LANGUAGE GADTs #-}
+module Language.Souffle.Interpreter
```

Comment:
There should be cleansing in the LANGUAGE pragmas section.
There should be more inline pragmas in this module.